docs/back-end-api/erc20CollateralToken/default.md

```diff
@@ -10,6 +10,100 @@ Outlined below is a detailed overview of the API's functionalities, leveraging a
 
 ## Endpoints List (GraphQL and RESTful API)
 
+### `Login`
+
+Return a JSON Web Token (JWT) access token and refresh.
```

Comment:
Specify the access token has a maturity of 60 minutes but it may vary depending on the server configuration.